--- /home/benjaminm/code/jedi-bundle/quenchxx/src/quenchxx/Interpolation.cc.tmp.bak	2025-01-18 06:59:04.664098981 +0100
+++ /home/benjaminm/code/jedi-bundle/quenchxx/src/quenchxx/Interpolation.cc	2025-01-18 06:58:09.132909138 +0100
@@ -13,6 +13,8 @@
 
 #include "oops/util/FieldSetHelpers.h"
 
+#include "quenchxx/VariablesSwitch.h"
+
 // -----------------------------------------------------------------------------
 
 namespace quenchxx {
@@ -86,7 +88,7 @@
     fset.haloExchange();
 
     // Apply unstructured interpolator
-    const oops::Variables vars(fset.field_names());
+    const varns::Variables vars(fset.field_names());
     std::vector<double> vals;
     unstructuredInterp_->apply(vars, fset, vals);
 
@@ -137,7 +139,7 @@
     }
 
     // Apply unstructured interpolator, adjoint
-    const oops::Variables vars(tgtFieldSet.field_names());
+    const varns::Variables vars(tgtFieldSet.field_names());
     unstructuredInterp_->applyAD(vars, srcFieldSet, vals);
 
     // Exchange FieldSet halo, adjoint
